### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by DonaldTsang** [DangerOnTheRanger/maniwani#114]
1. Tab icon blinks/changes to alert thread activity
2. Tab title has a number with parentheses indicating number of new posts or "slid" thread

**Comment by DangerOnTheRanger** [comment]
Why not use the [Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/notification) that's a part of modern web browsers instead?

**Comment by DonaldTsang** [comment]
@DangerOnTheRanger more compact and more permanent IMO.
Notifications API tends to be over-developed and obnoxious from a UX perspective.
Same can't be said about mobile, where notifications is all the rage

**Comment by DangerOnTheRanger** [comment]
I disagree that proper notifications are more obnoxious; if you don't want a site to use them, you have the option of not allowing them via permissions - there's no similarly-easy method of preventing a site from modifying its title.

**Comment by DonaldTsang** [comment]
@DangerOnTheRanger the problem of having to see a notification asking about whether to keep notification, is itself a UX blunder (similarly to how the GDPR notice makes things annoying). There is no way of disabling it in the browser to tell the website to NEVER show any notification or even a remote hint of "having" notifications (since every website is not the same).
Assume that the majority of the user does not want it in the first place, and then have a button in the settings panel to allow for it. If it is done the opposite way, then it would alienate the masses. Frictionless experience is key, notifications (and other form of "blinking noise") is just bad UX that breeds a bad attention economy.

**Comment by DangerOnTheRanger** [comment]
>There is no way of disabling it in the browser to tell the website to NEVER show any notification or even a remote hint of "having" notifications (since every website is not the same).

At least [Chrome](https://support.google.com/chrome/answer/3220216?co=GENIE.Platform%3DDesktop&hl=en) and [Firefox](https://blog.mozilla.org/firefox/no-notifications/) allow disabling notification requests on a per-site basis (including immediately after the site in question first requests notification permissions, and remember that you will see no notifications until permission is granted), and I imagine Edge and others have similar functionality. If you stick to actually using the Notifications API for everything notification-related, it's a pretty simple matter from the perspective of the user to allow your site to send notifications or not.

>Frictionless experience is key, notifications (and other form of "blinking noise") is just bad UX that breeds a bad attention economy.

I agree with this statement, but for this very reason I think the Notifications API makes more sense to use. If a user doesn't want notifications, they simply click "no" on the permissions dialog and never have to deal with it again. Messing with the document title made sense before the API got standardized and supported, but we have much better, more universal ways of conveying notifications now that everyone on the web has come to know how to work with.
